.palantir/revapi.yml

```diff
@@ -320,38 +320,6 @@ acceptedBreaks:
         \ org.apache.iceberg.types.Types.StructType, java.util.function.BiFunction<org.apache.iceberg.types.Type,\
         \ java.lang.Object, java.lang.Object>)"
       justification: "Removing deprecations for 1.2.0"
-    - code: "java.method.removed"
```

Comment:
I am trying to understand why are we removing these now. 

Was it because it is still available to the `BaseTableScan` via parent class so it is not really removed?

Reply:
oh, I think I get it now. 
rev API considered it as a breaking change because API was changed to package private. Now API is changed back to `protected`. So, In this version no interface change.  Hence, removed those entires.

Reply:
yes, RevAPI considered it as a breaking change when visibility changed (because subclasses in different packages wouldn't have access to that method anymore)